_posts/2020-08-06-changelog-128.markdown

```diff
@@ -0,0 +1,171 @@
+---
+layout: post
+title: "IntelliJ Rust Changelog #128"
+date: 2020-08-06 12:00:00 +0300
+---
+
+
+## New Features
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5715 -->
+* Improve `New project` dialog to support `cargo-generate` templates.
+Now a WebAssembly project can be created using the corresponding template (by [@avrong])
+
+    {% include gif-img.html path="/assets/posts/changelog-128/cargo-generate" w="700px" %}
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5451 -->
+* Add intention to import a complex path.
+Now you can add `use` item for any path under the caret (by [@Kobzol])
+
+    {% include gif-img.html path="/assets/posts/changelog-128/import-path" w="500px" %}
+
+* `Move` refactoring (<kbd>F6</kbd>) improvements:
+    <!-- https://github.com/intellij-rust/intellij-rust/pull/5736 -->
+    * Search and update changed references when moving a top-level item
+        
+        {% include gif-img.html path="/assets/posts/changelog-128/move-update-references" w="600px" %}
+
+    <!-- https://github.com/intellij-rust/intellij-rust/pull/5484 -->
+    * Take reexports into account when moving a single file
+
+        {% include gif-img.html path="/assets/posts/changelog-128/move-reexports" w="600px" %}
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5758 -->
+* Provide `true`/`false` completion for boolean values (by [@ozkriff]). N
+
+    {% include gif-img.html path="/assets/posts/changelog-128/bool-completion" w="700px" %}
+
+* Improve `Wrong number of type arguments` inspection
+    <!-- https://github.com/intellij-rust/intellij-rust/pull/5811 -->
+    * Support trait objects (by [@Kobzol])
+
+    <!-- https://github.com/intellij-rust/intellij-rust/pull/5306 -->
+    * Provide `Add missing type arguments` fix (by [@Kobzol])
+
+        {% include gif-img.html path="/assets/posts/changelog-128/missing-type-arg" w="700px" %}
+            
+* Improve struct patterns completion:
+    <!-- https://github.com/intellij-rust/intellij-rust/pull/5660 -->
+    * Suggest only struct fields in struct pattern completion (by [@Kobzol])
+    
+        {% include gif-img.html path="/assets/posts/changelog-128/struct-fields-comp" w="700px" %}
+
+    <!-- https://github.com/intellij-rust/intellij-rust/pull/5825 -->
+    * Completion for `..` syntax in struct patterns (by [@Kobzol])
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5808 -->
+* Better type inference for `associated_type_bounds` rustc nightly feature
```

Comment:
nit: Oh, it's not "Better" because we didn't suppport this feature before this change